client/src/main/scala/com/github/dapperware/slack/SlackRequests.scala

```diff
@@ -26,6 +26,16 @@ trait SlackRequests {
         .response(asJson[Json])
     )
 
+  def requestQueryParams(method: String, params: Seq[(String, String)]): UIO[Request[SlackResponse[Json], Any]] = {
+    val uri = uri"https://slack.com/api/$method"
+      .addParams(params:_*)
+    UIO.succeed(
+      basicRequest
+        .post(uri)
+        .response(asJson[Json])
+    )
+  }
```

Comment:
I don't think I've seen this variation in the docs before, it is usually one of the two above (if not json then it will have x-www-urlencoded).

Reply:
added header `content-type` as `application/json`